compose.yaml

```
@@ -0,0 +1,15 @@
+services:
+
+  backend:
+    build: .
+    container_name: backend
+    ports:
+      - "8000:8000"
+    volumes:
+      - .:/usr/src/novo-site
```

```suggestion
      - .:/var/www/html
```



O workdir padrão do container oficial do PHP, que é o que estamos utuilizando neste PR, é `/var/www/html`. Se montarmos o volume para `/var/www/html` não vai precisar criar um workdir customizado e segue o padrão do container oficial.

https://hub.docker.com/_/php